src/main/java/net/earthcomputer/multiconnect/impl/PacketSystem.java

```diff
@@ -34,10 +34,9 @@ private PacketSystem() {}
             try {
                 clazz = Class.forName("net.earthcomputer.multiconnect.generated.Protocol_" + protocol.getName().replace('.', '_'));
             } catch (ClassNotFoundException e) {
-                // TODO: make this a hard error once the packet system is complete
-                LOGGER.warn("Protocol class not found for {}", protocol.getName());
-                continue;
+                throw new IllegalStateException(String.format("Protocol class not found for %s", protocol.getName()));
```

Comment:
This prevents the packet system from starting at the moment because not all protocols are implemented yet